build_tools/android/run_benchmarks.py

```diff
@@ -99,9 +83,10 @@ def adb_push_to_tmp_dir(content, relative_dir, verbose=False):
   - The full path to the content on the Android device.
   """
   filename = os.path.basename(content)
-  android_path = f"{ANDROID_TMP_DIR}/{relative_dir}/{filename}"
-  execute(["adb", "push", os.path.abspath(content), android_path],
-          verbose=verbose)
+  android_path = os.path.join(f"{ANDROID_TMP_DIR}", f"{relative_dir}",
+                              f"{filename}")
```

Comment:
Can't this be `os.path.join(ANDROID_TMP_DIR, relative_dir, filename)`? Do we need f-string here?

Reply:
Actually yes. I didn't think much when converting this. :)